spinnaker_camera_driver/cmake/download_spinnaker

```diff
@@ -37,47 +40,140 @@ import glob
 
 logging.basicConfig(level=logging.INFO)
 
-LOGIN_URL = 'https://www.ptgrey.com/login'
-LOGIN_DATA = {
-    'Email': 'code@clearpathrobotics.com',
-    'Password': 'uNjRxoH6NMsJvi6hyPCH'
-    }
+URL_TEMPLATE = URL_TEMPLATES = {
+    'deb': 'https://packages.clearpathrobotics.com/stable/flir/Spinnaker/Ubuntu{version}/spinnaker-1.27.0.48-Ubuntu{version}-{arch}-pkg.tar.gz',
```

Comment:
If we're now in the clear to host this, all of this can be dropped in favour of asking OSRF to pull the debs onto their mirror. Make a PR here:

https://github.com/ros-infrastructure/reprepro-updater/tree/master/config

Reply:
The packages are not currently hosted on an actual deb server.  There's some question about whether or not it's worth setting one up to host them vs the immediate this-works-the-same-as-it-always-has downloading shenanigans.
Also, for the PointGrey camera driver (https://github.com/ros-drivers/pointgrey_camera_driver), the armhf and arm64 archives do not contain deb packages at all, so what you're suggesting isn't an option for that combination of package & architectures.

Reply:
Fair. If we don't want to go to the trouble of packaging the ARM assets for OSRF to host, another option would be to do a one-time unpack of this stuff and commit the headers and binary blobs into this repo (or script it so that that step can be easily rerun later on when Spinnaker gets updated).

I just know that in general, accessing external network resources during build is considered an anti-pattern, and this specific instance of it has caused issues in the past both with the public buildfarm and OTTO's bundle builder.